lib/cluster.js

```diff
@@ -0,0 +1,24 @@
+const qs = require('querystring');
+const Fetch = require('./fetch');
+const { httpBody } = require('./response-transforms');
+
+const info = (conn, params) => {
+  const headers = conn.headers();
+  headers.set('Accept', 'application/json');
+  return Fetch.fetch(conn.request('admin', 'cluster'), {
+    headers,
+  }).then(httpBody);
+};
+
+const status = (conn, params) => {
+  const headers = conn.headers();
+  headers.set('Accept', 'application/json');
+  return Fetch.fetch(conn.request('admin', 'cluster', 'status'), {
+    headers,
+  }).then(httpBody);
+};
+
+module.exports = {
+  info,
+  status,
+};
```

Comment:
I changed the import to make it possible to mock the fetch function. Since it needs an Object to spyOn, couldn't get it to work with a direct import. But I'm working on getting the cluster to run with docker compose and then I can change the tests to a full integration instead of mocks to address this.

Reply:
I was able to get a simple cluster to run and have updated the tests to use the cluster nodes for the connection without mocking fetch. This should address the code style issues.